include/boost/geometry/algorithms/detail/overlay/get_turn_info.hpp

```diff
@@ -46,10 +46,10 @@ class turn_info_exception : public geometry::exception
         message += method;
     }
 
-    virtual ~turn_info_exception() throw()
+    virtual ~turn_info_exception()
     {}
```

Comment:
We can better remove these constructors and destructors, in most cases we don't define them.

Reply:
Maybe we can merge this as is?
I will do the cleanup later on.

Reply:
You probably have to change it anyway, because the commit mixes extensions and normal code...
But of course, indeed, we could do the cleanup in a separate PR tool.

Reply:
I do not understand the problem with the extensions directory.

Could you, please, explain it to me?

Reply:
Btw, it is unclear if these can be removed or defaulted.

This is more like a codestyle consideration.

Reply:
Indeed, it is. I think they can all be removed, don’t see what it adds.

About `extensions`: we often merge from develop to master by cherry-picking commits. Master does not contain anything from the `extensions` folder. So then cherry-picking does not work or needs manual editing.